test/beforeRequest.test.js

```diff
@@ -606,43 +539,51 @@ describe("test validating response", () => {
             test(`reloading off after sign: ${element.name}`, () => {
                 let before;
                 let after;
+                const commStruct = {};
+                commStruct[commVersion] = {L: G, H: H};
+                const newUrl = new URL(CAPTCHA_HREF + EXAMPLE_SUFFIX);
```

Comment:
This seems to be duplicated in line 559.

Reply:
Now line 499 (https://github.com/privacypass/challenge-bypass-extension/pull/93/files#diff-3e403d715f0cc2c440d7fe415e6e5373R499)